package.json

```diff
@@ -16,9 +16,10 @@
     "format": "prettier --write",
     "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx .",
     "lint-css": "stylelint \"src/**/*.css\"",
-    "link-check": "./scripts/link-check-git-all.sh",
-    "link-check-diff": "./scripts/link-check-git-diff.sh",
-    "link-check-exclude": "./scripts/exclude-links-check.sh"
+    "link-check": "repo-link-check -c='./.linkcheckrc.json' -s='filesystem' -u",
```

Comment:
hmm ... why `=` though? it's not a very common convention for CLI, just `-c ./.linkcheckrc.json` should be enough

Reply:
That's just how I happened to write it, I can make that change and I'm sure `minimist` can handle it.

Reply:
yeah, I would prefer to have it simpler if possible. thanks!